### PR TITLE
IDEMPIERE-6309 Output of task is better formatted as a console font

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-6309

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] I have checked my code and corrected any misspellings
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
### Tests
- [x] I have tested the direct scenario that my code is solving
- [x] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WTask.java [hunk 1]
```diff
@@ -123,10 +123,12 @@ public void run() {
...
 						try {
 							StringBuilder sb = new StringBuilder();
-							sb.append(osTask.getOut())
+							sb.append("<pre>")
+							.append(osTask.getOut())
 							.append("<br>-----------<br>")
 							.append(osTask.getErr())
-							.append("<br>-----------");
+							.append("<br>-----------<br>")
+							.append("</pre>");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WTask.java:131]: hmm .. how is that ensure the output is using console font ?

**Comment by hengsin** [comment]
The usual web font family for console font: https://www.w3schools.com/css/tryit.asp?filename=trycss_font-family (see .p3)

**Comment by hengsin** [comment]
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre , monospace is typically use for pre but it is not something guarantee by any means.

**Comment by CarlosRuiz-globalqss** [comment]
> According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre , monospace is typically use for pre but it is not something guarantee by any means.

Hi @hengsin - I tried different options for fonts for the following script:
[test.sh.txt](https://github.com/user-attachments/files/17849954/test.sh.txt)

The best about showing the information in firefox and vivaldi was with \<pre\>\<code\> (this was also the recommendation from chatgpt)

It looks like this:

<kbd>test</kbd>

Other combinations of fonts didn't show the format as expected.

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**